lib/kafo/kafo_configure.rb

```diff
@@ -39,6 +40,7 @@ def hooking
     end
 
     def initialize(*args)
+      self.class.preset_color_scheme
```

Comment:
since this happens before the logger is built, could you make the color scheme used at https://github.com/theforeman/kafo/blob/master/lib/kafo/logger.rb#L95 so we don't duplicate ARGV parsing logic?